### ISSUE TITLE
See if it is feasible to use some kind of SVG optimizer when adding new icons

### ISSUE THREAD

**Issue by Elanhant** [gatsby-inc/gatsby-interface#55]
Currently adding a new icon includes a bit of manual work to clean up the SVG code. Perhaps it would make sense to use a tool like [SVGO](https://github.com/svg/svgo) to automate this.

**Comment by DSchau** [comment]
Yes please! I typically just paste into the web UI, which while it works, is not ideal!
